src/core/network.js

```diff
@@ -74,7 +74,7 @@ if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('FIREFOX || MOZCENTRAL')) {
       // blocked, e.g. via the connect-src CSP directive or the NoScript addon.
       // When this error occurs, this feature detection method will mistakenly
       // report that moz-chunked-arraybuffer is not supported in Firefox 37-.
-      x.open('GET', 'https://example.com');
+      x.open('GET', window.location.href);
```

Comment:
The code in the `src/core/` folder is executed in a worker, where `window` is not defined. Thus accessing `window` from within a worker will throw an error, which will short-circuit the logic below and cause `supportsMozChunked` to incorrectly always be `false`.

I'm not sure if we want to change to current code here, but if so then using `self` instead of `window` might be a possible solution.